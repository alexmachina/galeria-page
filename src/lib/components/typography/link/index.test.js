import React from "react";
import renderer from "react-test-renderer";
import Link from ".";

describe("Components", () => {
  describe("Link", () => {
    it("Renders", () => {
      const tree = renderer
        .create(<Link to="google.com.br">Google it!</Link>)
        .toJSON();

      expect(tree).toMatchSnapshot();
    });
  });
});
