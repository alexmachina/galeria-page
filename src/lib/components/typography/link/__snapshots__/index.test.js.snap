// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components Link Renders 1`] = `
<a
  className="link-0-2-1"
  href="google.com.br"
>
  Google it!
</a>
`;
